src/styles/main.scss

```
@@ -1,3 +1,28 @@
 body {
   margin: 0;
 }
+
+.snake {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+
+  &__item {
+    color: white;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 100px;
+    text-align: center;
+    line-height: 300px;
+  }
+
+  @for $i from 1 through 6 {
+    &__item:nth-child(#{$i}) {
```

```suggestion
    &__item:nth-child(#{$i}) {
```
Я задавал цвет модификаторами